core/domain/app_feedback_report_domain_test.py

```diff
@@ -243,6 +311,30 @@ def test_report_scrubber_id_is_invalid_validation_fails(self) -> None:
             'The scrubbed_by user id \'%s\' is invalid.' % (
                 self.android_report_obj.scrubbed_by))
 
+    def test_validate_android_report_with_bot_id_passes(
+        self
+        ) -> None:
+        """Tests that validation passes 
+        when the scrubbed_by ID is the bot ID.
```

Comment:
I think you can move more of this docstring to the previous line.

Reply:
It was causing lint errors edited the comment  
"Tests that validation passes when the scrubbed_by ID is the bot ID."
 -> "Tests that validation passes when the scrubbed_by ID is bot ID."